extensions/data-transfer/portability-data-transfer-flickr/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2018 The Data Transfer Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+log4j.rootLogger=DEBUG, A1
```

Comment:
Do we want to keep a log4j file in this directory?

Reply:
I'm not sure how the log4j is configured per java jar - When running unit tests, I got an error that the logger wasn't initialized and made debugging difficult so I added it in. 

Since I'm no longer using the logs, I'll revert this file.